### DISCUSSION TITLE
Sentry unsupported

### DISCUSSION THREAD

**Discussion by sgoodluck** [cloudflare/next-on-pages#178] [Q&A]
Potentially related: https://github.com/cloudflare/next-on-pages/issues/174

The logging tool, Sentry, seems to be incompatible with next-on-pages. 
At least, when following their recommended documentation. 

I suspect the incompatibility has to do with the _error component it generates. 
NextJS _error page ref: https://nextjs.org/docs/advanced-features/custom-error-page

Is there a better way to do this where Sentry will be supported?
Has anyone successfully configured Sentry with a next deployment to CFP?

**Comment by dario-piotrowicz** [comment]
@sgoodluck sorry don't really use Sentry, but I was wondering, have you tried this solution? https://github.com/cloudflare/next-on-pages/issues/174#issuecomment-1511398486

**Reply by sgoodluck** [reply]
Yes indeed / that's our present workaround

**Reply by dario-piotrowicz** [reply]
I see, I'm glad that you could go that route :slightly_smiling_face: 

I am sorry that it's clearly not ideal, but I am not sure if we can do anything about it unless Vercel fixes/changes their implementation :disappointed:

**Reply by simpson** [reply]
I was just about to post this same finding and now I'm sad because this was posted back in April... I pinged Vercel on the other [ticket](https://github.com/vercel/next.js/issues/48490)

**Reply by dario-piotrowicz** [reply]
yeah... unfortunately the issue there isn't really getting much attention 😓
